crates/oracle/src/main.rs

```diff
@@ -85,7 +85,7 @@ async fn main() -> Result<(), Error> {
                     tokio::time::sleep(sleep_time).await;
                     continue;
                 }
-                Continue(None) => continue,
```

Comment:
This `continuing` was short-circuiting the loop, which should go to sleep after a recoverable error.